src/components/AppStyle.tsx

```diff
@@ -88,6 +93,11 @@ export const AppThemeContext = createContext(theme);
 const CssContainer = styled.div`
   @import url("https://fonts.googleapis.com/css2?family=Lato&display=swap");
 
+  @font-face {
+    font-family: LatoExtended;
+    src: url(Lato-Regular.ttf);
```

Comment:
I think this needs the woff2,woff,eot in order of precedence.  Also see https://web.dev/articles/reduce-webfont-size#web_font_formats

Reply:
Done, thank you!